com.woltlab.wcf/templates/commentList.tpl

```diff
@@ -1,102 +1,144 @@
 			data-can-edit="{if $comment->isEditable()}true{else}false{/if}" data-can-delete="{if $comment->isDeletable()}true{else}false{/if}"
 			data-responses="{@$comment->responses}" data-last-response-time="{if $ignoreLastResponseTime|empty}{@$comment->getLastResponseTime()}{else}1{/if}" data-is-disabled="{@$comment->isDisabled}"
 			data-last-response-id="{if $ignoreLastResponseTime|empty}{@$comment->getLastResponseID()}{else}0{/if}"
 		>
-			<div class="box48{if $__wcf->getUserProfileHandler()->isIgnoredUser($comment->userID, 2)} ignoredUserContent{/if}">
-				{user object=$comment->getUserProfile() type='avatar48' ariaHidden='true' tabindex='-1'}
-				
-				<div class="commentContentContainer" itemprop="comment" itemscope itemtype="http://schema.org/Comment">
-					<div class="commentContent">
+			<woltlab-core-comment comment-id="{@$comment->commentID}" itemprop="comment" itemscope itemtype="http://schema.org/Comment">
+				<div class="comment__header">
+					<div class="comment__avatar">
+						{user object=$comment->getUserProfile() type='avatar48' ariaHidden='true' tabindex='-1'}
+					</div>
+					<div class="comment__author" itemprop="author" itemscope itemtype="http://schema.org/Person">
+						{if $comment->userID}
+							<a href="{$comment->getUserProfile()->getLink()}" class="userLink" data-object-id="{@$comment->userID}" itemprop="url">
+								<span itemprop="name">{@$comment->getUserProfile()->getFormattedUsername()}</span>
+							</a>
+						{else}
+							<span itemprop="name">{$comment->username}</span>
+						{/if}
+					</div>
+					<div class="comment__date">
 						<meta itemprop="dateCreated" content="{@$comment->time|date:'c'}">
+						<a href="{$comment->getLink()}">{@$comment->time|time}</a>
+					</div>
+					<div class="comment__status">
+						{if $comment->isDisabled}
+							<span class="badge label green comment__status--disabled">{lang}wcf.message.status.disabled{/lang}</span>
+						{/if}
 						
-						<div class="containerHeadline">
-							<h3 itemprop="author" itemscope itemtype="http://schema.org/Person">
-								{if $comment->userID}
-									<a href="{$comment->getUserProfile()->getLink()}" class="userLink" data-object-id="{@$comment->userID}" itemprop="url">
-										<span itemprop="name">{@$comment->getUserProfile()->getFormattedUsername()}</span>
+						{if $commentManager->isContentAuthor($comment)}
+							<span class="badge label">{lang}wcf.comment.objectAuthor{/lang}</span>
+						{/if}
+					</div>
+					<div class="comment__menu dropdown" id="commentOptions{@$comment->commentID}">
+		                <button type="button" class="button small dropdownToggle" aria-label="{lang}wcf.global.button.more{/lang}">{icon name='ellipsis-vertical'}</button>
+
+		                <ul class="dropdownMenu">
+		                    {if $comment->isDisabled && $commentCanModerate}
+								<li>
+									<a href="#" class="comment__option comment__option--enable">
```

Comment:
Should this be a `<button>` instead? Likewise below.

Reply:
A `<button>` would be better here but `dropdownMenu` doesn't support the `button` element yet.